test/e2e/audit.go

```diff
@@ -44,14 +44,26 @@ var _ = describe("Audit", func() {
 	})
 
 	It("Should audit API calls to create, update, patch, delete pods. [Audit] [Zalando]", func() {
+		runAsNonRoot := true
 		pod := &apiv1.Pod{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: "audit-pod",
 			},
 			Spec: apiv1.PodSpec{
 				Containers: []apiv1.Container{{
 					Name:  "pause",
-					Image: "container-registry.zalando.net/teapot/pause:3.4.1-master-18",
+					Image: "container-registry.zalando.net/teapot/pause:3.7-master-21",
+					// Drop all capabilities, run as non-root, and use the default seccomp profile.
+					// This is required as pods created are assumed to be restricted by default since k8s v1.31.
+					SecurityContext: &apiv1.SecurityContext{
+						Capabilities: &apiv1.Capabilities{
+							Drop: []apiv1.Capability{"ALL"},
+						},
+						RunAsNonRoot: &runAsNonRoot,
```

Comment:
Let's try to use `ptr.To(true)` using the `k8s.io/utils/ptr` package like above.

Reply:
I just dropped the context and the test worked, so we don't need this I guess.